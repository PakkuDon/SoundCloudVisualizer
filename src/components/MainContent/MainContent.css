@import "../../styles/colors";

.root {
  background-color: $black;
  flex: 3 0 auto;
  display: flex;
  flex-direction: column;
  align-items: center;
}
