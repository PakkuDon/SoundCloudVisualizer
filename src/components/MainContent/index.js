import React from 'react'
import PropTypes from 'prop-types'

const MainContent = ({ children }) => (
  <div id="body">
    {children}
  </div>
)

export default MainContent
