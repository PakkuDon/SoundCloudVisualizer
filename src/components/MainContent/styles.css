.root {
  background-color: #000;
  flex: 3 0 auto;
  display: flex;
  flex-flow: column nowrap;
  align-items: center;
  position: relative;
}
