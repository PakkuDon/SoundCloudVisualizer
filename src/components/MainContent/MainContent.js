import React from "react"
import PropTypes from "prop-types"
import styles from "./MainContent.css"

const MainContent = ({ children }) => (
  <div className={styles.root}>{children}</div>
)

MainContent.propTypes = {
  children: PropTypes.node,
}

export default MainContent
