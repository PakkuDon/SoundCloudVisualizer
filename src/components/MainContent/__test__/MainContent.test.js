import React from "react"
import { shallow } from "enzyme"

import MainContent from "../MainContent"

describe("MainContent", () => {
  it("renders children", () => {
    const children = <div>Hello world</div>
    const wrapper = shallow(<MainContent>{children}</MainContent>)

    expect(wrapper.contains(children)).toBe(true)
  })
})
