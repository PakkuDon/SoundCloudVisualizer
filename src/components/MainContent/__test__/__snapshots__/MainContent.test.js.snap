// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContent renders children 1`] = `
<div
  className="root"
>
  <div>
    Hello world
  </div>
</div>
`;
